Assets/MixedRealityToolkit/Inspectors/ExtensionTemplates/ExtensionServiceWizard.cs

```diff
@@ -0,0 +1,370 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    public partial class ExtensionServiceWizard : EditorWindow
```

Comment:
Why is this a partial class?

Reply:
Relic from earlier iteration, will remove.